cmd/importer/README.md

```diff
@@ -20,14 +20,47 @@ go build  -C cmd/importer/ -o $(pwd)/bin/importer
 
 The command runs against the systems default kubectl configuration. Check the [kubectl documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to learn more about how to Configure Access to Multiple Clusters.
 
+### Check
 The importer will perform following checks:
 
 - At least one `namespace` is provided.
-- The label key (`queuelabel`) providing the queue mapping is provided.
-- A mapping from one of the encountered `queuelabel` values to an existing LocalQueue exists.
+- For every Pod a  mapping to a LocalQueue is available.
+- The target LocalQueue exists.
 - The LocalQueues involved in the import are using an existing ClusterQueue.
 - The ClusterQueues involved have at least one ResourceGroup using an existing ResourceFlavor. This ResourceFlavor is used when the importer creates the admission for the created workloads.
 
+The are two ways the mapping from a pod to a local queue can be specified:
+
+#### Simple mapping
+
+It's done by specifying a label name and any number of <label-value>=<localQueue-name> a command line arguments eg.  `--queuelabel=src.lbl --queuemapping=src-val=user-queue,src-val2=user-queue2`.
+
+#### Advanced mapping
+
+It's done providing an yaml mapping file name as `--queuemapping-file` argument, it's expected content being:
+
+```yaml
+- match:
+    labels:
+      src.lbl: src-val
+  toLocalQueue: user-queue
+- match:
+    priorityClassName: p-class
+    labels:
+      src.lbl: src-val2
+      src2.lbl: src2-val
+  toLocalQueue: user-queue2
+- match:
+    labels:
+      src.lbl: src-val3
+  skip: true
+```
+
+- During the mapping, if the match rule has no `priorityClassName` the `priorityClassName` of the pod will be ignored, if more than one `label: value` pairs are provided, all of them should match.
+- The rules are evaluated in order.
+- `skip: true` can be used to ignore the pods matching a rule.
```

Comment:
could the user leave a rule with no priorityClassName and no labels with `skip:true`?

That could be a way of saying: if it doesn't match any rule above, I just don't care about this pod.

Reply:
yes,

```yaml
-skip: true
```

at the end is doing that.

similarly 
```yaml
-toLocalQueue: user-queue2
```
will put all the pods in one localQueue